lib/game/game_manager.ex

```diff
@@ -21,6 +21,22 @@ defmodule Reactor.GameManager do
     GenServer.call(@name, {:get_games})
   end
 
+  def get_users(game_id) do
+    game_id
+    |> to_reg
+    |> GenServer.call({:get_users})
+  end
+
+  def add_user_to_game(game_id, user) do
+    game_id
+    |> to_reg
+    |> GenServer.cast({:add_user, user})
```

Comment:
I think this should probably be a `call` not a `cast` as you're more than likely going to want to know the result